src/generators/bendable-amogus/AmongUsBendableGenerator.res

```diff
@@ -0,0 +1,254 @@
+let requireImage = fileName => Generator.requireImage("./images/" ++ fileName)
+let requireTexture = fileName => Generator.requireImage("./textures/" ++ fileName)
+
+let id = "bendableamogusgenerator"
```

Comment:
Let's rename the `id` to `among-us-bendable`

Reply:
Done